adc.js

```diff
@@ -33,13 +33,49 @@ define([], function () {
     };
 
     Adc.prototype.onComplete = function () {
-        var val = 0x8000;
-        // TODO: switch on bottom two bits of adc_status and pick a value corresponding
-        // to the appropriate axis.
+      var val = 0x8000;
+		//val = Math.floor(Math.random() * 10000);
```

Comment:
Can you remove this comment, and indent the rest of this function consistently please? The rest of the codebase is using 4-space indents, not tabs. Cheers :)